.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+### What was wrong?
+
+Related to Issue #
+Closes Issue #
```

Comment:
I just noticed this from the template. I think this will actually close the issue as opposed to doing `Closes #`. We should double check but I would vote to change this here and in the template to `Closes #`.